spec/models/student_loans/eligibility_spec.rb

```diff
@@ -116,29 +129,64 @@
     end
   end
 
+  context "when saving in the “subjects-taught” context" do
+    it "is not valid if none of the subjects-taught attributes are true" do
+      expect(StudentLoans::Eligibility.new).not_to be_valid(:"subjects-taught")
+      expect(StudentLoans::Eligibility.new(biology_taught: false)).not_to be_valid(:"subjects-taught")
+      expect(StudentLoans::Eligibility.new(biology_taught: false, physics_taught: false)).not_to be_valid(:"subjects-taught")
+    end
+
+    it "validates the when one or more of the subjects-taught attributes are true" do
+      expect(StudentLoans::Eligibility.new(biology_taught: true)).to be_valid(:"subjects-taught")
+      expect(StudentLoans::Eligibility.new(biology_taught: true, computer_science_taught: false)).to be_valid(:"subjects-taught")
+      expect(StudentLoans::Eligibility.new(chemistry_taught: true, languages_taught: true)).to be_valid(:"subjects-taught")
+    end
+
+    it "validates with no subjects present if mostly_teaching_eligible_subjects is false" do
```

Comment:
Can we rephase "validates" to "is valid". I think that's easier to parse ("validates" refers to the process as well as the result).

Reply:
👍🏽